package.json

```diff
@@ -25,5 +25,9 @@
   "bugs": {
     "url": "https://github.com/apothuku/aclu-cash-bail/issues"
   },
-  "homepage": "https://github.com/apothuku/aclu-cash-bail#readme"
+  "homepage": "https://github.com/apothuku/aclu-cash-bail#readme",
+  "dependencies": {
+    "d3": "^6.2.0",
```

Comment:
Do we wanna include these dependencies as scripts? `<script src="https://d3js.org/d3.v6.min.js"></script>` ?

Reply:
Would we rather include dependencies as scripts, or bundle then with our JS? If we bundle with our JS, can we tree-shake to reduce the size of the dependency? Or is it more efficient for HTTP/2 to multiplex the script requests? I don't have experience in making these kinds of decisions.

Reply:
hmm not really sure about the efficiency of the two methods, but i think maybe it'll depends on what the webflow HTML/JS embed looks like. I'd assume that it'd just be easier to include dependencies in a script tag. Either way, I think we can figure this stuff out whenever we add stuff to webflow. this decision shouldn't block merging this branch.